app/assets/stylesheets/modules/sul-footer.scss

```diff
@@ -0,0 +1,55 @@
+#su-content {
+  padding-bottom: 200px !important;
+}
+
+#sul-footer-container {
+  padding: 0 !important;
```

Comment:
!important should not be used<br>Properties should be ordered background-color, -webkit-box-shadow, box-shadow, clear, margin-top, min-height, padding, position